app/routes/interestgroups/components/InterestGroupMemberList.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React, { Component } from 'react';
+import Modal from 'app/components/Modal';
+import styles from './InterestGroupMemberList.css';
+import { Link } from 'react-router';
+import { ProfilePicture } from 'app/components/Image';
+import Tooltip from 'app/components/Tooltip';
+import { Flex } from 'app/components/Layout';
+import Icon from 'app/components/Icon';
+import type { User, GroupMembership } from 'app/models';
+
+const iconName = (role: string) => {
+  switch (role) {
+    case 'leader':
+      return ['star', 'Leder'];
+    case 'co-leader':
+      return ['star-outline', 'Nestleder'];
+    default:
+      return;
+  }
+};
+
+const icon = (role: string) => {
+  const props = iconName(role);
+  if (props) {
+    const [name, tooltip] = props;
+    return (
+      <Tooltip content={tooltip}>
+        <Icon name={name} />
+      </Tooltip>
+    );
+  }
+};
+
+const ListedUser = ({ user, role }: { user: User, role: string }) => {
+  return (
+    <li>
+      <Flex className={styles.row}>
+        <ProfilePicture size={30} user={user} />
+        {icon(role)}
+        <Link to={`/users/${user.username}`}>{user.fullName}</Link>
+      </Flex>
+    </li>
+  );
+};
+
+const rolePriority = ['leader', 'co-leader'];
+
+type Props = {
+  children: any,
+  memberships: Array<GroupMembership>
+};
+
+type State = {
+  modalVisible: boolean
+};
+
+export default class InterestGroupMemberList extends Component<Props, State> {
+  state = { modalVisible: false };
+
+  toggleModal = () => {
+    this.setState(state => ({
+      modalVisible: !state.modalVisible
+    }));
+  };
+
+  render() {
+    this.props.memberships.sort((a, b) => {
```

Comment:
Maybe move this to a selector?

Reply:
You can also use the lodash `sortBy`, since the normal `sort` is inplace. Not that important tho.

Reply:
Or do it in the `componentWillMount`. https://github.com/webkom/lego-webapp/blob/master/app/components/UserAttendance/AttendanceModal.js#L47

Reply:
Is it ok to do it with lodash like [this](https://github.com/webkom/lego-webapp/blob/5736546e25aff5e4440e0ec04f223aee7a2bfb7d/app/routes/admin/groups/components/GroupMembersList.js#L22)?

Reply:
:+1: